crates/noirc_evaluator/src/ssa/acir_gen.rs

```diff
@@ -825,12 +828,17 @@ pub fn evaluate_and(
     let result = evaluator.add_witness_to_cs();
     let bsize = if bit_size % 2 == 1 { bit_size + 1 } else { bit_size };
     assert!(bsize < FieldElement::max_num_bits() - 1);
-    evaluator.gates.push(Gate::And(acvm::acir::circuit::gate::AndGate {
-        a: a_witness,
-        b: b_witness,
-        result,
-        num_bits: bsize,
-    }));
+
+    let gate = Gate::BlackBoxFuncCall(BlackBoxFuncCall {
+        name: acvm::acir::BlackBoxFunc::AND,
```

Comment:
Helper method to create AND with only one num_bits

Reply:
We can move this to an issue on acvm, since it has nothing to do with the migration

Reply:
I will put it under the heterogenous opcodes issue already opened in acvm